src/Sarif.UnitTests/SarifCurrentToVersionOneTests.cs

```diff
@@ -32,6 +32,45 @@ private static void VerifyCurrentToVersionOneTransformation(string v2LogText, st
             v1LogText.Should().Be(v1LogExpectedText);
         }
 
+        [Fact]
+        public void SarifTransformerTests_ToVersionOne_RestoreFromPropertyBag()
+        {
+            const string V2LogText =
+@"{
+  ""$schema"": ""http://json.schemastore.org/sarif-2.0.0"",
+  ""version"": ""2.0.0"",
+  ""runs"": [
+    {
+      ""tool"": {
+        ""name"": ""CodeScanner"",
+        ""semanticVersion"": ""2.1.0""
+      },
+      ""results"": [],
+      ""properties"": {
+        ""sarifv1/run"": ""{\""tool\"":{\""name\"":\""CodeScanner\"",\""semanticVersion\"":\""2.1.0\""},\""results\"":[]}""
```

Comment:
Without having yet read the code, it appears that when you persist a v1 `run` to the v2 `run`'s property bag, you compress out all the white space. When you extract the v1 `run` into its own log file, you pretty-print it, using `"\r\n"` as the newline sequence, and two-space indentation.

Another possible behavior would be to retain the original file's white space. The disadvantages of that approach are:
1. Increased v2 log file size.
2. Possible inconsistency between the `run` (which might use one whitespace convention) and the containing log file (which might use a different convention):
```
{
  "$schema": "",
  "version": "1.0.0",
  "runs": [                    # The v2-to-v1 converter generates everything up to here...
    {                              # ... but the extracted v1 run object, starting here, might use a different white space convention.
    }
  ]
}
```

All in all, I think you made the right choice, even though it doesn't preserve whitespace. But I'd like @michaelcfanning  to weigh in.